manager/testdata/notebook/README.md

```diff
@@ -10,7 +10,7 @@ are configured to use TLS:
 - Fybrik manager
 - Vault
 
-All servers above except Openmetadata server use mutual TLS.
+All servers above except Openmetadata and OPA server use mutual TLS. OPA server uses TLS. Openmetadata server not using TLS.
```

Comment:
Openmetadata server does not use TLS. 
or
Meantime, Openmetadata server does not support TLS.  (is it correct?)

Reply:
Fixed thanks. Last time I checked Openmetadata had some open issues related to their TLS support.